op-e2e/actions/garbage_channel_out.go

```diff
@@ -0,0 +1,223 @@
+package actions
+
+import (
+	"bytes"
+	"compress/gzip"
+	"compress/zlib"
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+var ErrNotDepositTx = errors.New("first transaction in block is not a deposit tx")
```

Comment:
Aren't these defined in `op-node`?

Reply:
Ah yep good shout- can de-duplicate these